### ISSUE TITLE
Interpret material core as MATERIAL SAMPLE occurrence

### ISSUE THREAD

**Issue by timrobertson100** [gbif/pipelines#885]
TLDR:

Accommodate DwC-A with [MaterialEntity Core](https://rs.gbif.org/sandbox/core/dwc_material_2023-04-29.xml) as if it were an Occurrence Core with a basisOfRecord of type `MaterialSample` using the `materialEntityID` field will uniquely identify the record. This may require a new dataset type in the Registry too 

**Background**

The work to refine the data model is moving toward support for the publication of datasets containing Material Entity records which, will often provide evidence of an Organism Occurrence - this may be the whole organism, a part of it, something left by it (e.g. scat) or a sample of the environment (e.g. soil) from which the organism can be detected (e.g. through DNA) etc. The conceptual model is fairly well progressed to accommodate this but has proven to be a complex endeavor for those involved in the [testing](https://github.com/gbif/model-material). 

We aim to explore two publishing models as approaches to address this, including a [Frictionless Data Package](https://github.com/gbif/rs.gbif.org/issues/106) and a [Material Entity Core](https://rs.gbif.org/sandbox/core/dwc_material_2023-04-29.xml) DwC-A with participants in the SPNCH workshop (End May 2023). To aid that workshop, we'd like the ability for datasets mapped to this DwC-A core in a TEST IPT to be indexed in UAT as Occurrence records - while it's not ideal to treat them as Occurrences it's known that seeing data indexed in GBIF UAT helps provide a sense of satisfaction for those attending - and it's all that is feasible in the timeframe. In the future, records of this type would be ingested into a Material Catalogue and linked as evidence (or part of a set of evidence) for an Organism Occurrence. Work on the Material Catalogue evolves ([current demo](http://labs.gbif.org:7003/)) but is not ready for this workshop.

Does this capture what we discussed, please @tucotuco?

CC @mike-podolskiy90 for IPT implications on registering in UAT from IPT

**Comment by tucotuco** [comment]
Yes, that captures what we discussed. We talked about a middle ground
between the two options highlighted here, which may have implications. We
want one track in SPNHC to be able to use the MaterialEntity Core with
extensions as one might do with an Occurrence Core. This might causes
glitches if the extensions don't declare that they can be attached to a
MaterialEntity Core. Is that the case in practice in the IPT?

**Comment by timrobertson100** [comment]
@marcos-lg [here](https://ipt3.gbif-uat.org/resource?r=tim-mat-dwca) is a single record dataset example with 1 image attached

**Comment by timrobertson100** [comment]
>  This might causes glitches if the extensions don't declare that they can be attached to a MaterialEntity Core. Is that the case in practice in the IPT?

Good point - tracking that [here](https://github.com/gbif/rs.gbif.org/issues/110)

**Comment by marcos-lg** [comment]
Deployed to UAT for testing.

Example dataset: https://registry.gbif-uat.org/dataset/f50e01bd-3c6f-44e0-a442-1b4411f97616
Example results: https://api.gbif-uat.org/v1/occurrence/search?datasetKey=f50e01bd-3c6f-44e0-a442-1b4411f97616

**Comment by MattBlissett** [comment]
@ymgan please see this issue for updates on material entity core ingestion.
